src/components/shared/video-card/VideoCardActionMenu.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import { MdThumbDown, MdThumbUp, MdVideoLibrary } from "react-icons/md";
+import { RiPlayList2Line, RiShareForwardLine } from "react-icons/ri";
+import { useLikedVideos } from "../../../context/liked-videos/LikedVideoContext";
+import { usePlaylist } from "../../../context/playlist/PlaylistContext";
+import PlaylistModal from "../../playlist/PlaylistModal";
+import DropdownMenu from "../DropdownMenu";
+import Modal from "../modal/Modal";
+
+const VideoCardActionMenu = ({ videoId, playlistId, isInLiked }) => {
+  const { addToLikes, removeFromLikes } = useLikedVideos();
+  const [showModal, setShowModal] = useState(false);
+  const { removeFromPlaylist } = usePlaylist();
+
+  const addToPlayList = (id) => {
+    if (!playlistId) setShowModal((v) => !v);
+    else removeVideoHandler(videoId, playlistId);
+  };
+
+  const likeDislikeClickHandler = () => {
+    if (isInLiked) removeFromLikes(videoId);
+    else addToLikes(videoId);
+  };
+
+  const removeVideoHandler = (videoId, playlistId) => {
+    if (videoId || playlistId) removeFromPlaylist({ playlistId, videoId });
+  };
+
+  return (
+    <>
+      <DropdownMenu>
+        <div
+          className="flex items-center gap-1 pointer"
+          onClick={() => console.log(videoId)}
```

Comment:
remove console.log if not necessary here

Reply:
Thank you, I will remove it.